src/components/Grants/GrantBeneficiaryItem.tsx

```diff
@@ -60,7 +62,7 @@ function GrantBeneficiaryItem({ grant }: Props) {
               <ProgressBarTooltip grant={grant} isInCliff={proposalInCliffPeriod}>
                 <div>
                   {proposalInCliffPeriod ? (
-                    <CliffProgress enactedAt={enacted_at} basic />
+                    <CliffProgress enactedAt={enacted_at!} basic />
```

Comment:
If enacted_at is necessary for CliffProgress and VestingProgress then I'd remove the `!` and move the render condition here.